### ISSUE TITLE
Does Batfish support layer3 EVPN?

### ISSUE THREAD

**Issue by sunjackj** [batfish/batfish#8834]
Does Batfish support layer3 EVPN?I want to work on some development based on Batfish recently, but it seems that Batfish does not support EVPN for Layer3?

**Comment by dhalperi** [comment]
Yes, Batfish supports some versions of EVPN, primarily on NX-OS. See, for example, https://pybatfish.readthedocs.io/en/latest/notebooks/vxlan_evpn.html?highlight=evpn#L3-EVPN-VNIs
